.travis.yml

```diff
@@ -9,6 +9,7 @@ services:
   - docker
 
 before_script:
+  - sudo /etc/init.d/postgresql stop
```

Comment:
travis runs it's own postgres and mysql.

Basically without this line we can't start our own postgres because port 5432 is already busy